recipes/calysto_bash/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "calysto_bash" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ba733e55979377c2f1a905a8fe3c04b20178c382c2a4072b45afa95d99890b40
+
+build:
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+    - {{ PYTHON }} -m calysto_bash install --sys-prefix
+
+requirements:
+  host:
+    - pip
+    - python
+    - ipykernel
+    - metakernel
+  run:
+    - python
+    - ipykernel
+    - jupyter_client
+    - metakernel
+    - bash  # [unix]
+    - m2-bash  # [win]
+
+test:
+  imports:
+    - calysto_bash
+  commands:
+    - jupyter kernelspec list | grep bash
+    - jupyter run --kernel=bash command.sh
+  files:
+    - command.sh
+
+about:
+  home: https://github.com/takluyver/bash_kernel
```

Comment:
this doesn't seem right either.

Reply:
Fixed, things like that happens when there is a baby crawling around you. :-)